src/test/resources/features/Meeds/MeedsUsr_Mng.feature

```diff
@@ -1,9 +1,9 @@
-# language: en
+@users
 Feature: Users Management
   As an administrator
   I want to check that i can manage the users
 
-  @ignored
+
   Scenario: [USR_MNG-1.1] Users Grid
     Given I am authenticated as admin
```

Comment:
please review the scenarios to not having to refresh the page multiple times while it's useless

Reply:
done